src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(/images/star.svg) center center;
+  background-repeat: no-repeat;
+  width: $star-size;
+  height: $star-size;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :nth-child(-n+1),
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
```

Comment:
SASS makes it possible to use the cycle "for" in such situations.

Reply:
agree. SASS loop would fit perfect here